src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -24,9 +25,9 @@ public final class KeyContainsMasker implements JsonMasker {
      */
     private static final int MIN_OFFSET_JSON_KEY_QUOTE = 3;
     /**
-     * Minimum JSON for which masking could be required is: {"":1}, so minimum length at least 6 bytes.
+     * Minimum JSON for which masking could be required is: [0], so minimum length at least 3 bytes.
      */
-    private static final int MIN_MASKABLE_JSON_LENGTH = 6;
+    private static final int MIN_MASKABLE_JSON_LENGTH = 3;
```

Comment:
We might as well remove this "optimization" 😄

Reply:
Agreed. At this point it might as well go 😅 

However, I vaguely remember it had two purposes: the "optimization" but also protected against some ArrayIndexOutOfBound because it was reading ahead. 

I don't think that still applies, but lets make sure :-)

Reply:
I will check if it can be removed